hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/ReadableAccountStoreImpl.java

```diff
@@ -81,6 +81,16 @@ public Account getAccountById(@NonNull final AccountID accountID) {
         return account == null ? null : account;
     }
 
+    @Override
+    @Nullable
+    public AccountID getAccountIDByAlias(@NonNull final String alias) {
+        final var entityNum = aliases.get(alias);
+        if (entityNum == null) {
+            return null;
+        }
+        return AccountID.newBuilder().accountNum(entityNum.num()).build();
+    }
```

Comment:
Instead of putting the same implementation in both `TestFixturesKeyLookup` and `ReadableAccountStoreImpl`, wouldn't it make more sense to have a `default` method implementation in the `ReadableAccountStore` interface?

Reply:
Ah yes, duh 🤦 👍